airflow/hooks/__init__.py

```diff
@@ -27,6 +27,7 @@
 }
 
 _import_module_attrs(globals(), _hooks)
+from airflow.contrib.hooks import *
```

Comment:
@mistercrunch got it. It's now accessible from `airflow.contrib.hooks.FTPHook` and I added a placeholder module to show to how to add operators to contrib.